test/plots/piePlotTests.ts

```diff
@@ -2,92 +2,140 @@
 
 describe("Plots", () => {
   describe("PiePlot", () => {
-    // HACKHACK #1798: beforeEach being used below
-    it("renders correctly with no data", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let plot = new Plottable.Plots.Pie();
-      plot.sectorValue((d) => d.value);
-      assert.doesNotThrow(() => plot.renderTo(svg), Error);
-      assert.strictEqual(plot.width(), 400, "was allocated width");
-      assert.strictEqual(plot.height(), 400, "was allocated height");
-      svg.remove();
-    });
+    describe("Rendering", () => {
+      let svg: d3.Selection<void>;
+      let simpleDataset: Plottable.Dataset;
+      let simpleData: any[];
+      let piePlot: Plottable.Plots.Pie;
 
-    it("each slice consists of a fill path and a stroke path", () => {
-      let svg = TestMethods.generateSVG(500, 500);
-      let piePlot = new Plottable.Plots.Pie();
-      piePlot.sectorValue((d) => d.value);
-      let data = [
-        { value: 1 },
-        { value: 1 }
-      ];
-      let dataset = new Plottable.Dataset(data);
-      piePlot.addDataset(dataset);
-      piePlot.renderTo(svg);
-      let renderArea = (<any> piePlot)._renderArea;
-      let arcPaths = renderArea.selectAll(".arc");
-      let arcFillPaths = renderArea.selectAll(".arc.fill");
-      let arcOutlinePaths = renderArea.selectAll(".arc.outline");
-      assert.strictEqual(arcPaths.size(), 4, "2 paths per datum");
-      assert.strictEqual(arcFillPaths.size(), 2, "1 fill path per datum");
-      assert.strictEqual(arcOutlinePaths.size(), 2, "1 outline path per datum");
-      assert.strictEqual(arcFillPaths.style("stroke"), "none", "fill paths have no stroke");
-      assert.strictEqual(arcOutlinePaths.style("fill"), "none", "outline paths have no fill");
-      svg.remove();
-    });
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(500, 500);
```

Comment:
Then let's compute the pixel positions based on the 500 extracted into `svgWidth` and `svgHeight`